services/jobrunner/justfile

```diff
@@ -52,3 +52,13 @@ jobs-stats:
 job-retry job_id:
     python3 -m jobrunner.cli.retry_job {{ job_id }}
 
+# deploy latest version of job-runner
+deploy:
+    #!/bin/bash
+    set -euo pipefail
+    cd ~openafely/jobrunner/code
```

Comment:
```suggestion
    cd ~opensafely/jobrunner/code
```

Reply:
Sorry, this is an old PR, didn't even know it existed, will close

This change got merged in #183